src/pyH2A/Plugins/Photovoltaic_Plugin.py

```diff
@@ -105,27 +122,86 @@ def calculate_H2_production(self, dcf):
 		yearly_data = []
 
 		for year in dcf.operation_years:
```

Comment:
New funtion fo the osmosis which is below: —› here from the H2-production for a year the demand_osmosis is calculated —› this is then abstracted from the power_consumption (?, the power which we can use for the hydrogen production) for the following year. So the calculation here is depending on the electricity from the previous year —› write new function for this and substract it on th part when needed